packages/shared/components/enums/input.enum.ts

```diff
@@ -0,0 +1,5 @@
+export enum InputType {
+    Text = 'text',
+    Number = 'number',
+    Password = 'password',
+}
```

Comment:
This enum should live inside of the component

Reply:
as we use this enum in more than one component, I think this folder is fine; however, I renamed it :wink: